### PR TITLE
Changelog cleanup

### PR DESCRIPTION
**Pull request by flihp**
This is preparation for the upcoming CHANGELOG update for the 2.0 release.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,17 @@ the 'common directory.
...
 
+## [1.4.0] - 2018-03-02
+### Added
+- Attached Component commands from the last public review spec.
+### Fixed
+- Essential files missing from release tarballs are now included.
+- Version string generation has been moved from configure.ac to the
+bootstrap script. It is now stored in a file named `VERSION` that is
+shipped in the release tarball.
+- We've stopped shipping the built man page for InitSocketTcti.3 and now
```
Comment (tstruk) [inline CHANGELOG.md]: Prebuilt man page for InitSocketTcti.3 replaced with the source.

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,6 +16,17 @@ the 'common directory.
 ### Fixed
 - Wrong return type for Tss2_Sys_Finalize (API break).
 
+## [1.4.0] - 2018-03-02
+### Added
+- Attached Component commands from the last public review spec.
```
Comment (tstruk) [inline CHANGELOG.md]: Attached Component commands from the last public review spec (rev 1.46)

**Comment by flihp** [comment]
I don't disagree with your proposed improvements / clarifications but this is the changelog entry that was in the 1.4.0 release tarball. I'm not sure changing these is really the right thing to do since it will cause future changelogs to disagree with past changelogs. In fact I'd prefer to leave these as is if you're not too invested in these changes.

**Comment by tstruk** [comment]
If this is how it is in 1.4.0 then sure, we can merge it as is.

**Pull request merged**